srcpkgs/chezmoi/update

```diff
@@ -0,0 +1,2 @@
+site=https://github.com/twpayne/chezmoi/releases/latest
+pattern='href="/twpayne/chezmoi/releases/tag/v\K[\d.]+(?=")'
```

Comment:
that is not necessary as tags are found on github by `xbps-src update-check chezmoi` just fine

Reply:
does it use the `distfiles` or `homepage` field for update - checking? From my understanding of the manual, it was `homepage`, which is not a github url in my template.